File: kotlin/FizzBuzz.kt
fun main() {
    out = "";
    for (i in 1..100)
        out += when {
            i % 15 == 0 -> "FizzBuzz "
            i % 3 == 0 -> "Fizz "
            i % 5 == 0 -> " Buzz "
            else -> "$i "
        }
    print(out)    
}

File: php/fizzBuzz.php
<?php
$out = '';
for ($i=1; $i <= 100; $i++) { 
    if ($i % 15 == 0) {
        $out .= 'FizzBuzz ';
    }
    elseif ($i % 3 == 0) {
        $out .= 'Fizz ';
    }
    elseif ($i % 5 == 0) {
        $out .= 'Buzz ';
    } else {
        $out .= $i . ' ';
    }
}
echo $out;